micropub.php

```diff
@@ -882,13 +882,36 @@ public static function default_file_handler( $post_id ) {
 	 * https://ownyourswarm.p3k.io/docs#checkins
 	 *
 	 * WordPress geo data is stored in post meta: geo_address (free text),
-	 * geo_latitude, and geo_longitude:
+	 * geo_latitude, geo_longitude, and geo_public:
 	 * https://codex.wordpress.org/Geodata
+	 * It is noted that should the HTML5 style geolocation properties of altitude, accuracy, speed, and heading are
+	 * used they would use the same geo prefix. Simple Location stores these when available using accuracy to estimate
+	 * map zoom when displayed.
 	 */
 	public static function store_geodata( $args ) {
 		$properties = static::get( static::$input, 'properties' );
 		$location   = static::get( $properties, 'location', static::get( $properties, 'checkin' ) );
 		$location   = static::get( $location, 0, null );
+		// Location-visibility is an experimental property https://indieweb.org/Micropub-extensions#Location_Visibility
+		// It attempts to mimic the geo_public
+		$visibility = static::get( $properties, 'location-visibility', null );
+		if ( $visibility ) {
+			$visibility = array_pop( $visibility );
+			if ( ! isset( $args['meta_input'] ) ) {
+				$args['meta_input'] = array();
+			}
+			switch ( $visibility ) {
+				case 'public':
+					$args['meta_input']['geo_public'] = 1;
```

Comment:
mind adding a comment with a link to wordpress docs about what these numeric values mean? or even better, are there constants in the wordpress code that we could use instead?

Reply:
It is in the codex link referenced. However, will comment about what they mean inline.